monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -43,6 +43,12 @@ def get_target_monkey_by_os(is_windows, is_32bit):
 
 def build_monkey_commandline_explicitly(parent=None, tunnel=None, server=None, depth=None, location=None):
     cmdline = ""
+    
+    from infection_monkey.config import WormConfiguration                              #Passing Multiple Servers for Monkey to connect back
+    Server_String =" "
+    for server in WormConfiguration.command_servers:
+         Server_String = Server_String + server + " "
+    server = Server_String
```

Comment:
Now that we know that it works lets move this code to it's proper place. Why can't we keep this code here:
1. Function should do one thing. `build_monkey_commandline_explicitly` builds monkey command line from given arguments. This code creates a server string from command servers in monkey configuration. 
2. [Pure functions](https://www.geeksforgeeks.org/pure-functions/)

Reply:
Do you suggest to move this code into a new function?